documentation/installation.md

```diff
@@ -79,11 +79,129 @@ the development server at http://localhost:5000/ using your web browser. Log in
 with the previously created superuser account credentials and you will then be
 redirected to the Ocim webpage which lists the Open edX instances.
 
-### Local installation (skip this if using Vagrant)
+### LXC installation
 
-If you prefer not to use Vagrant, you can install OpenCraft Instance Manager manually.
+If you're running Linux on your workstation and would like the added
+performance of running in a container, we recommend using
+[LXD](https://linuxcontainers.org/lxd/) to provision your development
+environment.  This will keep your development environment isolated from the
+rest of your system just like the Vagrant devstack, but with better performance
+and flexibility.
+
+LXD is a wrapper over [LXC](https://linuxcontainers.org/lxc/) that makes
+container management much simpler.  You will also use Ansible
+[Ansible](https://www.ansible.com/) to provision your container.
+
+The rest of this section assumes you're running a recent version of Ubuntu.
+
+Start by opening a terminal and setting some useful environment variables.
+`SRC` should point to the OCIM checkout, and `CONTAINER` is the desired LXC
+container name.
+
+    SRC=~/src/ocim
+    CONTAINER=ocim
+
+Check the necessary repositories out:
+
+    mkdir -p $SRC
+    cd $SRC
+    git clone https://github.com/open-craft/opencraft ocim
+    git clone https://github.com/open-craft/ansible-playbooks
```

Comment:
@arbrandes We need to clone the playbooks inside the Ocim folder anyway, so why not do it in this step?
See the vagrant instructions here: https://github.com/open-craft/opencraft/blame/master/documentation/installation.md#L35

Reply:
For what its worth, that's not true for LXC, where you don't run the playbooks from bare metal.  You run them from within the container itself.